src/Calendar.js

```diff
@@ -102,7 +102,8 @@ let Calendar = React.createClass({
      *   slotInfo: object {
      *     start: Date,
      *     end: Date,
-     *     slots: array<Date>
+     *     slots: array<Date>,
+     *     action: string
```

Comment:
can you change this to `"select" | "click"` so it's clearer what these values are?